lahja/tools/benchmark/utils/config.py

```diff
@@ -0,0 +1,17 @@
+from typing import (
+    Tuple,
+)
+
+from lahja import (
+    ConnectionConfig,
+)
+
+
+def create_consumer_endpoint_configs(num_processes: int) -> Tuple[ConnectionConfig, ...]:
+    return tuple(
+        ConnectionConfig.from_name(create_consumer_endpoint_name(i)) for i in range(num_processes)
```

Comment:
Minor but it seems simpler to just inline the f-string:

```python
ConnectionConfig.from_name(f"consumer_{i}")
```

Reply:
Yes but we use that at another place and I wouldn't want two places to maintain a `f"consumer_{i}" thing.